### PR TITLE
Renamed classes to correspond with library name

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@arcturus r?

File: src/main/java/org/mozilla/magnet/scanner/BaseScanner.java [hunk 1]
```diff
@@ -1,8 +1,5 @@
-package org.mozilla.magnet.net.scanner;
```
Comment (arcturus) [inline src/main/java/org/mozilla/magnet/scanner/BaseScanner.java]: Leave the `net` package, or rename it to `network`.

If later on we need to do more native libraries we will add them to another namespace like `org.mozilla.magnet.notifications`

**Comment by arcturus** [comment]
Renaming looks good, just want to keep the package `net`

r+!

File: src/main/java/org/mozilla/magnet/scanner/BaseScanner.java (hunk 1 above)
Comment (wilsonpage) [inline src/main/java/org/mozilla/magnet/scanner/BaseScanner.java]: I don't regard bluetooth scanning as 'network' scanning. Namespacing `scanner` suggests that we may someday have more than one android scanner package. I don't think this is likely.

**Pull request merged**